action.yaml

```diff
@@ -0,0 +1,28 @@
+name: 'Run Iter8 Experiment'
+description: 'Run an Iter8 experiment'
+author: Iter8
```

Comment:
I think its perfectly ok to so "Michael Kalantar" here  (along with the email you use to sign off). Its up to you. Iter8 authors is also perfectly ok.